src/components/validation/runner/config.rs

```diff
@@ -34,18 +36,21 @@ impl TopologyBuilder {
                 "No test case name defined for configuration {:?}.",
                 config_name
             ))?;
+
+        let log_namespace = configuration.log_namespace == LogNamespace::Vector;
```

Comment:
I'm curious why you carry this around as a `bool` instead of the original `LogNamespace`?